### ISSUE TITLE
AIO header navigation bar ARIA and screen-reader support

### ISSUE THREAD

**Issue by twerske** [angular/angular#44335]
### Description

Lighthouse report issue is: `List items (<li>) are not contained within <ul> or <ol> parent elements.`

In theory this navigation should be a nav element with a nested `<ul>` and each item as an `<li>` and nested `<a>` tags. The role="navigation" is overriding the role="list" invalidating the screen reader's ability to navigate.

<img width="676" alt="Screen Shot 2021-12-01 at 2 19 08 PM" src="https://user-images.githubusercontent.com/15061394/144323296-684d23fa-b8d6-4368-86e8-bc8a5337aec8.png">

**Comment by dario-piotrowicz** [comment]
@twerske I believe I fixed this as part of #45209 (sorry I missed this issue)
